### PR TITLE
Issue 52

### PR DESCRIPTION
**Pull request by VAO11**
@womenhackfornonprofits/whfnp
## Linked Jira/Github Issue Ticket

Issue - #52
## What are the changes included in this PR?

Updated the buttons on the projects page
- view open source project
- visit website
  ## If the changes are visual, provide a screenshot.

*Desktop View:"
1 - whfnp_projs-btns_ 52

2 - whfnp_projs-btns2_ 52

_Mobile View:_
1 - whfnp_projs-m-btns2_ 52
2 - whfnp_projs-m-btns_ 52

### CODE REVIEW

File: src/projects.html [hunk 1]
```diff
@@ -100,20 +72,12 @@ <h2 class="section-heading">Jaspals Voice</h2>
...
-                the next few years.</p><a class="button project-info"
-                href="https://github.com/womenhackfornonprofits/pimp-my-cause-django">View
-                Pimp My Cause Project</a>
+                <p>Pimp My Cause connects marketers, researchers, and innovators who wish to volunteer to good causes in need of innovative solutions. The organisation has successfully matchmade over 1,300 charities and volunteers since its inception in
+                    2011.</p>
+                <p>WHFNP volunteers are tackling a website rebuild project for this charity to modernise the technology stack and improve the site's user experience. Using Django, Sass and JavaScript, the end goal is to leave Pimp My Cause with a slick new
+                    website that is robust, sustainable, and capable of supporting the growth in its membership expected over the next few years.</p>
+                <a class="button project-info" href="https://github.com/womenhackfornonprofits/pimp-my-cause-django">View Open Source Project</a>
+                <a class="button project-info" href="https://staging-pimpmycause.herokuapp.com/">Visit website</a>
```
Comment (lili2311) [inline src/projects.html:80]: can we make the buttons smaller? they are too wide at a glance. Can we not just use the standard buttons? Why do they need to be customised?

Comment (VAO11) [reply src/projects.html:80]: Hi @lili2311, thanks for the comments. I actually didn't change the size of the button. It is the same as what is already available in CSS.

Only added a hidden class to hide buttons that don't have a link. Also, standardised the text on the buttons.

I can customise them if you want. Did you like what Laura did in PR #124?

Comment (lili2311) [reply src/projects.html:80]: yes please let's use standard button only and remove the extra code and class. Also if a button doesn't have a ink just delete the html for now. No need to hide with css if thats what you mean.

Comment (lili2311) [reply src/projects.html:80]: Just the standard buttons is fine, so both pink as normal.

Comment (lili2311) [reply src/projects.html:80]: did you have a chance to make the changes please?

Comment (lili2311) [reply src/projects.html:80]: or we can just merge this and make a new issue to tidy up later, what do you think?

**Comment by lili2311** [comment]
👍

**Pull request merged**